utils/sshclient.go

```diff
@@ -150,7 +150,7 @@ func (c *Client) ScriptFile(fname string) *RemoteScript {
 // A RemoteScript represents script that can be run remotely.
 type RemoteScript struct {
 	client     *ssh.Client
-	_type      remoteScriptType
```

Comment:
sorry, i should say more clearly, i mean the name _type should be changed

Reply:
I see.
Thank you.